src/Hl7.Fhir.Serialization/FhirXmlNode.cs

```diff
@@ -437,5 +437,38 @@ object checkRepresentation(ITypedElement node, IExceptionSource ies, object _)
 
         private ExceptionNotification buildException(string message) => ExceptionNotification.Error(
                 new StructuralTypeException("Parser: " + message));
+
+        public string OriginalElement
+        {
+            get {
+                if (((XElement)Current.Parent).Name.Namespace != ((XElement)Current).Name.Namespace)
+                {
+                    return ((XElement)Current).ToString(SaveOptions.DisableFormatting);
+                }
+                else
+                {
+                    var test = StripNamespaces((XElement)Current);
```

Comment:
not really a nice variable name `test`

Reply:
I should start naming things seriously even when I testing around. Chagend this to: stripedXElement